index.html

```diff
@@ -1,23 +1,71 @@
 <!DOCTYPE html>
 <html lang="en">
 
-  <head>
-    <meta charset="utf-8">
-    <meta name="description" content="Your description goes here">
-    <meta name="keywords" content="one, two, three">
+<head>
+  <meta charset="utf-8">
+  <meta name="description" content="Your description goes here">
+  <meta name="keywords" content="one, two, three">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,600,800" rel="stylesheet">
 
-    <title>Blank Template</title>
 
-    <!-- external CSS link -->
-    <link rel="stylesheet" href="css/normalize.css">
-    <link rel="stylesheet" href="css/style.css">
+  <title>Relaxr</title>
 
-  </head>
+  <!-- external CSS link -->
+  <link rel="stylesheet" href="css/normalize.css">
+  <link rel="stylesheet" href="css/style.css">
 
-  <body>
-    <div class="container">
-      <!-- Replace me with your html -->
+</head>
+
+<body>
+  <div class="container">
+    <!-- Replace me with your html -->
+    <h1> Relaxr</h1>
+    <h2> Get a piece of mind with a single tap</h2>
+    <div class="push">
+
+      <button class="click" type="submit">Get it Now
+
+      </button>
     </div>
-  </body>
+  </div>
+
+  <div class="benefits">
+    <h3>Benefits</h3>
+    The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.
+
+    <ul>
+      <li>Schdule meetings for you</li>
+      <li>Excel automation to complete your work for you</li>
+      <li> Responds to emails on your behalf</li>
+      <li> Does all your work for you with our revolutionary A1 technology</li>
+    </ul>
+  </div>
+  <div class="happy quote">
+    <p>"“Relaxr changed my life. I’ve been able to travel the world, spend limited time working and my boss keeps thanking me
+      for crushing work."</p>
+
+    <h3>- Amanda, Intuit </h3>
+  </div>
+  <div class="push joke">
+    <button class="pop" type="submit"> Sign up Now!</button>
+  </div>
+
+
+
+
+
+</body>
```

Comment:
The last two tags in **every** HTML file should be your closing </body> and closing </html>. All of your HTML goes inside the <body></body> tags. No exceptions.